src/grouped-legend-styles.ts

```diff
@@ -15,19 +15,53 @@ export class Styles {
     };
   }
 
+  /**
+   * positon: top
+   * - groupedlegend-global-container: flex-direction: column
+   * - groupedlegend-legend-container: order: 0
+   *
+   * position: bottom
+   * - groupedlegend-global-container: flex-direction: column
+   * - groupedlegend-legend-container: order: 1
+   *
+   * position: left
+   * - groupedlegend-global-container: flex-direction: row
+   * - groupedlegend-legend-container: order: 0
+   *
+   * position: right
+   * - groupedlegend-global-container: flex-direction: row
+   * - groupedlegend-legend-container: order: 1
+   */
+
+  get globalContainer(): Partial<CSSStyleDeclaration> {
```

Comment:
I don't have a very good idea of this container, I don't know if it contains just the legend, or both the legend and the chart. If it is the former, the `flexGrow: 1` looks very sus, and might be the cause of the following 50/50 split on the verticality between the legend and the chart.
![image](https://user-images.githubusercontent.com/91089839/224345213-6620e460-b4cd-4d14-b645-6536464531ac.png)